### PR TITLE
[10.0] add module access_control

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: access_control/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```
Comment (jbchouinard) [inline access_control/__openerp__.py]: __manifest__.py?

File: access_control/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# © 2011 Smile
+# © 2017 Savoir-faire Linux
+# License GPL-3.0 or later (http://www.gnu.org/licenses/gpl).
+
+{
+    "name": "Access Control",
+    "version": "8.0.1.0.0",
```
Comment (jbchouinard) [inline access_control/__openerp__.py]: 10

File: access_control/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "author": "Smile",
+    "website": 'http://www.smile.fr',
+    "license": 'AGPL-3',
```
Comment (jbchouinard) [inline access_control/__openerp__.py]: les en-têtes disent GPL

File: access_control/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2011 Smile
+# © 2017 Savoir-faire Linux
+# License GPL-3.0 or later (http://www.gnu.org/licenses/gpl).
+
+{
+    "name": "Access Control",
+    "version": "8.0.1.0.0",
+    "author": "Smile",
```
Comment (jbchouinard) [inline access_control/__openerp__.py]: SFL?

**Review by jbchouinard — CHANGES_REQUESTED** [review]
I would ask to update the manifest the rest looks ok

File: access_control/__openerp__.py (hunk 3 above)
Comment (ghost) [reply access_control/__openerp__.py]: fixed

File: access_control/__openerp__.py (hunk 4 above)
Comment (ghost) [reply access_control/__openerp__.py]: fixed

File: access_control/__openerp__.py (hunk 2 above)
Comment (ghost) [reply access_control/__openerp__.py]: fixed

File: access_control/__openerp__.py (hunk 1 above)
Comment (ghost) [reply access_control/__openerp__.py]: fixed

**Review by jbchouinard — APPROVED** [review]

**Pull request merged**